handler/__tests__/payload_subscribe.js

```diff
@@ -0,0 +1,74 @@
+const { Expect } = require("../../lib/testing");
+const facebook = require("../../lib/facebook");
+const payloadSubscribe = require("../payload_subscribe");
+
+const tableName = process.env.DYNAMODB_SUBSCRIPTIONS;
+
+describe("payload_subscribe.subscribe", () => {
+  // dynamodb: c3a20f370fae4beafdde6af472ff8bf63da9ef9b
+  it("adds appropriate labels and replies with the correct text", () => {
+    const chat = new facebook.Chat({sender: {id: "1"}});
+    return payloadSubscribe.subscribe(chat, {subscription: "morning"}).then(() => {
+      new Expect(chat)
+      .labelAdded('push-breaking')
+      .labelAdded('push-morning')
+      .text(`Ich schick dir ab jetzt die Nachrichten, wie du sie bestellt hast. ` +
+        `Wenn du die letzte Ausgabe sehen willst, schreib einfach "Leg los"`);
+    });
+  });
+
+  it("adds a subscription to dynamodb", () => {
+    const chat = new facebook.Chat({sender: {id: "1"}});
+    return payloadSubscribe.subscribe(chat, {subscription: "morning"}).then(() => {
+      new Expect(chat).dynamoPut({
+        TableName: tableName,
+        Item: {
+          psid: "1",
+          morning: 1,
+          evening: 0
+        },
+        ConditionExpression: 'attribute_not_exists(psid)',
+      });
```

Comment:
Same matter as above: Sometimes I want to check that something was added to dynamodb without having a chat object. Is this considered?